scribejava-apis/src/test/java/com/github/scribejava/apis/examples/OdnoklassnikiExample.java

```diff
@@ -18,14 +18,14 @@
 
     public static void main(String... args) {
         // Replace these with your client id and secret
-        final String clientId = "your client id";
-        final String publicKey = "your api secret";
-        final String clientSecret = "your client secret";
```

Comment:
Why? What is secretKey in terms of OAuth2? There is client_secret, but no secret_key.